packages/react/src/redux.ts

```diff
@@ -121,7 +121,7 @@ function createReduxEnhancer(enhancerOptions?: Partial<SentryEnhancerOptions>):
         /* Action breadcrumbs */
         const transformedAction = options.actionTransformer(action);
         if (typeof transformedAction !== 'undefined' && transformedAction !== null) {
-          scope.addBreadcrumb({
+          addBreadcrumb({
```

Comment:
Changed this because this was setting on the scope, which would not have triggered the hook 😬 an example for how this _could_ miss a breadcrumb. The good thing is this is non-critical, meaning if we miss a breadcrumb it's not the end of the world, it's just not visible in the Replay.